shopping/CHANGES.md

```diff
@@ -1,6 +1,16 @@
 # Changes
 
 
+## [1.3.0](https://github.com/googleapis/google-cloud-go/releases/tag/shopping%2Fv1.3.0) (2025-11-06)
```

Comment:
Why is this a minor bump to `1.3.0` when the change types are only fixes and docs?

Reply:
All changes that are generated by Librarian from API changes end up as minor bumps, as we're not confident that we can apply the conventional commit types from googleapis... the same commit can be logically a chore in one language and a fix in another.